builder/vmware/common/driver_player_unix.go

```diff
@@ -115,7 +115,7 @@ func playerNetmapConfPath() string {
 
 func playerVerifyVersion(version string) error {
 	if runtime.GOOS != "linux" {
-		return fmt.Errorf("The VMWare Player version %s driver is only supported on Linux, and Windows, at the moment. Your OS: %s", version, runtime.GOOS)
+		return fmt.Errorf("driver is only supported on linux, not %s", runtime.GOOS)
```

Comment:
While this is the UNIX driver, I would maybe suggest that Windows is also supported here, the driver implementation likely has a windows-specific implementation, but from a user perspective using Windows would do the trick I imagine.

Reply:
Resolved in 67054cd.